### PR TITLE
test(tests/e2e): move client commands tests to a new suite

### PR DESCRIPTION
**Pull request by andrii-zakurenyi**

### CODE REVIEW

**Comment by andrii-zakurenyi** [comment]
oh, permissions, will figure out that later

**Comment by sheldonhull** [comment]
@andrii-zakurenyi do we want to keep this around? It's drifting from main and 2 weeks old. Not a big deal just making sure.

File: docs/developer/testing-end-to-end.md [hunk 1]
```diff
@@ -14,85 +14,21 @@ dsv user create --username <username> --password <password>
 
 ### Grant Permissions
 
-Create policies which will allow actions required for testing to the created user.
+This E2E testing suite covers most of the commands and actions available in the CLI.
+Therefore it is more convinient to add the newly created user to the list of admins of the tenant.
```
Comment (sheldonhull) [inline docs/developer/testing-end-to-end.md]:
```suggestion
It is more convenient to add the newly created user to the list of admins of the tenant.
```

File: docs/developer/testing-end-to-end.md [hunk 2]
```diff
@@ -14,85 +14,21 @@ dsv user create --username <username> --password <password>
...
-    --resources "roles:e2e-cli-test<.*>" \
-    --effect allow \
-    --actions "<create|read|update|delete>" \
-    --subjects users:<username>
-```
-
-- List all Roles
-
-```shell
+```bash
```
Comment (sheldonhull) [inline docs/developer/testing-end-to-end.md]:
```suggestion
```shell
```

Use shell instead of bash.

**Review by sheldonhull — APPROVED** [review]
Minor pending changes to apply

**Pull request merged**